interface/forms/eye_mag/SpectacleRx.php

```diff
@@ -446,7 +446,7 @@ function delete_me(delete_id) {
                 ?>
                     <div id="RXID_<?php echo attr($row['id']); ?>"
                          style="position:relative;text-align:center;margin: 10 auto;">
```

Comment:
Styling Error: use `position-relative` and `text-center` BS4 classes to replace inline styling here